src/Redux/water/waterReducer.js

```diff
@@ -0,0 +1,159 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import water from 'components/TodayWaterList/water.json';
+
+const INITIAL_STATE = {
+  waterArr: water || [],
+  isLoading: false,
+  error: null,
+  totalWaterAmmount: 0,
+  todayNorma: 1500,
+};
+
+// DEL WATER
+export const delWaterThunk = createAsyncThunk(
+  'todayWater/delete',
+  async (waterId, thunkAPI) => {
+    try {
+      return waterId;
+    } catch (e) {
+      return thunkAPI.rejectWithValue(e.message);
+    }
+  }
+);
+
+// CHENGE WATER
```

Comment:
chAnge

Reply:
as i know , in thunk we  could not jump to 'return ' without any Statement . You should to write comment which of api call will be here